tools/PI/DevHome.PI/BarWindowHorizontal.xaml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<winex:WindowEx
+    x:Class="DevHome.PI.BarWindowHorizontal"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:winex="using:WinUIEx"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevHome.PI.Controls"
+    xmlns:helpers="using:DevHome.PI.Helpers"
+    mc:Ignorable="d"
+    Title="" MinHeight="70" MinWidth="700" Height="70" Width="700" MaxHeight="70"
+    TaskBarIcon="Images/pi.ico"
+    Closed="WindowEx_Closed">
+
+    <!-- TODO Make DesktopAcrylicBackdrop/MicaBackdrop and AcrylicBackgroundFillColorBaseBrush user-configurable.-->
+    <Window.SystemBackdrop>
+        <MicaBackdrop/>
+    </Window.SystemBackdrop>
+
+    <Grid x:Name="MainPanel" Padding="0" Loaded="MainPanel_Loaded" Background="Transparent" SizeChanged="MainPanel_SizeChanged">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+
+        <!--Chrome buttons for a horizontal bar -->
+        <Grid x:Name="HorizontalBar" Background="Transparent" VerticalAlignment="Center" Height="28" Grid.Row="0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition x:Name="LeftPaddingColumn" Width="0"/>
+                <ColumnDefinition x:Name="ExtraSystemButtons" Width="*"/>
+                <ColumnDefinition x:Name="RightPaddingColumn" Width="0"/>
+            </Grid.ColumnDefinitions>
+
+            <TextBlock x:Name="TitleTextBlock" Text="DevHome PI" FontSize="10" Margin="10,0,0,0" HorizontalAlignment="Left" Grid.Column="1" VerticalAlignment="Center"/>
+            <StackPanel x:Name="ChromeButtonPanel" Orientation="Horizontal" VerticalAlignment="Center" HorizontalAlignment="Right" Margin="0, 0, 10, 0"  Background="Transparent" Grid.Column="1">
+                <Button
+                      x:Uid="SnapButton"
+                      x:Name="SnapButton" Style="{StaticResource ChromeButton}" FontSize="11"
+                      Click="{x:Bind viewModel.PerformSnapCommand}" HorizontalAlignment="Left" IsEnabled="{x:Bind viewModel.IsSnappingEnabled, Mode=OneWay}">
```

Comment:
Suggest either leaving all attributes on one line (for small number of attributes) or each one gets its own line (like here).

Reply:
I've been a fan of keeping it all on one line (feels easier to see the layout of all the controls, with the drawback of needing to scroll to see all of the controls properties). Is the feedback to just be consistent?